.github/workflows/push-tests.yml

```
@@ -75,26 +75,67 @@ jobs:
         with:
           registry: ${{ secrets.ECR_REPO }}
       - uses: actions/checkout@v2
+      - name: Get changed frontend or backend files
+        id: changed-files
+        uses: tj-actions/changed-files@v23.1
+        with:
+          files: |
+            frontend/src/**
+            frontend/tests/**
+            frontend/playwright*
+            frontend/package*
+            frontend/next*
```

Added a few more files I think we need to watch 😄 Thanks!

```suggestion
            frontend/next*
            frontend/playwright*
            frontend/package*
            frontend/doc-site*
            frontend/Dockerfile
            frontend/Makefile
            frontend/container_init.sh
            frontend/stylelint.config.js
            frontend/tsconfig.json
```

---

What files that are not included in this allow list, but are in `/frontend`, do we expect to result in high #'s of “erroneous” test runs? Is it worth having such a verbose allow-list?

---

Hmm good question. Basically you're asking if we should just watch `frontend/*`?

I guess any file changes that don't affect the build we could skip E2E tests? Like *.md. Maybe there's something else 🤔

---

I was under the impression we don't need to run e2e tests on doc site updates, Makefile updates, lint styling config update

Either way, FWIW, I'm going to wait on further work on this PR regardless because we'll need to refactor it for the portal redesign anyway and that structure is going to simplify this workflow

---

Ah yeah we currently don't have e2e tests for the doc site, but I was hoping to run the tests still, so when we add doc site tests, it will trigger e2e tests. But that could be over engineering on my end!

I added Makefile to it in case when command changes and break e2e, we'll catch it in the PR

Lint config changes probably don't need to trigger tests, since it only matters when the lint affects actual production code. So my bad for including it 😆

---

There's still some .mdx changes that can make it through the linter and still break `next build`. I think its worth testing on those in the meantime.